doc/install_generic.inc

```diff
@@ -91,6 +91,10 @@ libgpuarray
 Install the latest, development version of libgpuarray following the
 `Step-by-step instructions <http://deeplearning.net/software/libgpuarray/installation.html#step-by-step-install>`__.
 
+Alternatively, we also provide a conda channel ``mila-udem`` which contains packages for ``pygpu`` and ``libgpuarray``
+that are compatible with the latest development version of Theano::
```

Comment:
Currently, you need pygpu 0.7.1 that isn't in conda default channel. But you can install it with our own channel (that support only Python 2.7 and 3.5):

conda install -c mila-udem pygpu
pip install [--user] github... # For the latest dev version
pip install [--user] github... # For the beta2 version